R/configureApp.R

```diff
@@ -143,7 +142,7 @@ unsetProperty <- function(propertyName, appPath = getwd(), appName = NULL,
   # resolve the application target and target account info
   target <- deploymentTarget(appPath, appName, NULL, NULL, account)
   accountDetails <- accountInfo(target$account)
-  lucid <- lucidClientForAccount(accountDetails)
+  lucid <- clientForAccount(accountDetails)
```

Comment:
nit: These "client" instances are named `lucid`. I'm fairly sure that these functions apply to shinyapps.io and posit.cloud (they're both hosted by the "lucid" code-base, but that's a company-internal name). We could just call them `client`.